.travis.yml

```diff
@@ -1,19 +1,21 @@
 language: ruby
-sudo: false
+
 services:
   - postgresql
   - mysql
 bundler_args: --without tools benchmarks
+before_install:
+  - gem install bundler -v'< 2'
```

Comment:
Why is this needed?

Reply:
Oh, perhaps it was just a gut-reaction to the _gray badge_ not having built without an error. I'll try removing this.

Reply:
Ah, I remebered wrong: the issue is that the shipped version of Bundler can be too early to understand the platform name "truffleruby".

MRIs 2.3.8, 2.4.5, 2.5.3 come with Bundler 1.16.2.

Reply:
I got lucky with 2.